examples/http-server/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+)
+
+func corazaGetRequest(getPath string) (int, string) {
```

Comment:
I think it is good you abstract this but maybe you want to setup the test server in another function and then the tests can be read like:

```
testServer := setupTestServer()
defer testServer.Close()

statusCode := doHTTPCall(testServer.URL + getPath)
if statusCode != expectedStatusCode {
		log.Fatalln(errors.New("Returned status code:" + fmt.Sprint(statusCode) + ". Expected: " + strconv.Itoa(expectedStatusCode)))
	}
```